src/main/python/analyzers/project_analyzer.py

```diff
@@ -8,7 +9,14 @@ def __init__(self, tweetsAndStatsDF):
 
     def getSentimentAnalysis(self, tweetsList: list.__class__, regression=True):
         resultsDF = sa.getSentimentAnalysis(tweetsList)
-        # Combine resultsDF with self.tweetsAndStats.copy()
+        print("Results\n", resultsDF.index)
+        mergedDF = pd.merge(
```

Comment:
No idea if this works. Wasn't able to test because I could never get a response from azure.